package-windows-amd64/start.ps1

```diff
@@ -0,0 +1,4 @@
+Set-PSDebug -Trace 1
+Get-NetAdapter | New-NetIPAddress -IPAddress 169.254.169.251 -PrefixLength 32
+Start-Sleep 5
+C:\rancher-dns.exe $args
```

Comment:
I would add a `-never-recurse-to=169.254.169.251` argument to prevent a feedback loop in the recursion if `getGlobalRecurse()` picks up that IP as nameserver.

See Linux entrypoint: https://github.com/rancher/rancher-dns/blob/c1cf02535a785cd0f1d0631c2b1aaba88cb56a01/package/rancher-entrypoint.sh#L30